Nix/Parser.hs

```diff
@@ -56,8 +56,9 @@ nixHasAttr term = build <$> term <*> optional (reservedOp "?" *> nixSelector) wh
   build t Nothing = t
   build t (Just s) = Fix $ NHasAttr t s
 
-nixOpArg :: Parser NExpr
-nixOpArg = nixSelect $ choice
+-- | A self-contained unit.
+nixTerm :: Parser NExpr
```

Comment:
I just renamed this because I felt it was a little clearer (also it reflects the `term` nomenclature used in the `nixExpr` function)